### PR TITLE
Unit 12: Ethics and Bias, Conclusion

### PR DESCRIPTION
**Pull request by snehilsanyal**
Happy new year everyone🥳🤗 
Added the final chapter for Unit 12. Ethics and Bias. This chapter concludes the unit, and mentions Hugging Face's efforts for Ethics and Society.

**References:** [HF Society and Ethics Newsletter](https://huggingface.co/blog/ethics-soc-1) and [Society and Ethics Org](https://huggingface.co/society-ethics)
**Final chapter for the issue:** #51

### CODE REVIEW

**Comment by charchit7** [comment]
Flow looks good to me. 
Also, the space you mentioned by @merveenoyan , https://huggingface.co/spaces/merve/dataset-worldviews
It's really good! I came across via this PR only. Thanks!

**Comment by snehilsanyal** [comment]
> Flow looks good to me. Also, the space you mentioned by @merveenoyan , https://huggingface.co/spaces/merve/dataset-worldviews It's really good! I came across via this PR only. Thanks!

Yes it's an amazing space, the visuals and the way the content was arranged was awesome. I always thought how we could possibly deliver content on ethics and bias without being too harsh or insensitive about it. The space gave me cool ideas 🤗

File: chapters/en/Unit 12 - Ethics and Biases/conclusion.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+1. [**Socratic Models Image Captioning**](https://huggingface.co/spaces/Geonmo/socratic-models-image-captioning-with-BLOOM)
+2. [**Comparing Image Captioning Models**](https://huggingface.co/spaces/nielsr/comparing-captioning-models)
+
+### 🌎 Sustainable
+
+This is work that highlights and explores techniques for making machine learning ecologically sustainable. Some examples:
+- Tracking emissions from training and running inferences on large language models.
+- Quantization and distillation methods to reduce carbon footprints without sacrificing model quality.
+
+1. [**EfficientFormer**](https://huggingface.co/spaces/adirik/efficientformer)
```
[the remark this answers is not in the document]
Comment (snehilsanyal) [reply chapters/en/Unit 12 - Ethics and Biases/conclusion.mdx:64]: Yup I think some issues, will check.

File: chapters/en/Unit 12 - Ethics and Biases/conclusion.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    a. [🤗 *for Education Project*](https://huggingface.co/blog/education) aims at educating people from all backgrounds, beginners and instructors. Various experts and team members organize meetups, conferences and workshops.  
+    b. [*Data Measurement Tool*](https://huggingface.co/spaces/huggingface/data-measurements-tool) is an interactive interface and open-source library that lets dataset creators and users automatically calculate metrics that are meaningful and useful for responsible data development. 
+
+## Categories of Hugging Face Spaces 
+
+As we move forward, let us now take a look into how spaces are categorized by Hugging Face. Hugging Face categorizes the spaces in 6 high levels based on the ethical aspects in Machine Learning. Hugging Face Spaces are categorized as:
+
+### ✍️ Rigorous
+
+Rigorous projects pay special attention to examining failure cases, protecting privacy through security measures, and ensuring that potential users (technical and non-technical) are informed of the project's limitations. Some examples:
```
[the remark this answers is not in the document]
Comment (snehilsanyal) [reply chapters/en/Unit 12 - Ethics and Biases/conclusion.mdx:25]: So there are some more spaces, but I thought of including spaces that were focused on CV, if others suggest I will add more spaces.

**Review by merveenoyan — APPROVED** [review]
I think this is very nice and wholesome!

**Review by ATaylorAerospace — APPROVED** [review]
This is. great chapter @snehilsanyal!

**Comment by snehilsanyal** [comment]
Thank you @merveenoyan and @ATaylorAerospace for your quick reviews 🤗

**Pull request merged**